docs/prysm-usage/parameters.md

```diff
@@ -113,3 +114,24 @@ These flags are specific to launching a validator client.
 | ------------- |:-------------|
 |`--interop-start-index` | The start index to deterministically generate validator keys when used in combination with `--interop-num-validators`. Example: `--interop-start-index=5 --interop-num-validators=3` would generate keys from index 5 to 7.
 |`--interop-num-validators` | The number of validators to deterministically generate validator keys when used in combination with `--interop-num-validators`. Example: `--interop-start-index=5 --interop-num-validators=3` would generate keys from index 5 to 7.
+
+## Slasher parameters
+These flags are specific to launching a validator client.
+### Base flags
+| Flag          | Usage         |
+| ------------- |:-------------|
+|`--beacon-rpc-provider` | Specify the beacon node RPC provider endpoint (default: "localhost:4000")
+|`--beacon-tls-cert` | Provide the certificate for secure beacon gRPC connection. Pass this in order to use beacon gRPC securely.
+|`--rebuild-span-maps` | Rebuild span maps from indexed attestations in db (default: false)
+|`--rpc-port` | Specify the RPC port exposed by the slasher (default: 5000)
+|`--tls-key` | Specify the private key for secure gRPC. Pass this and the tls-cert flag in order to use gRPC securely.
+
+### Debug flags
+| Flag          | Usage         |
+| ------------- |:-------------|
+|`--cpuprofile` | Write CPU profile to the given file
+|`--memprofilerate` | Turn on memory profiling with the given rate (default: 524288)
+|`--pprof` | Enable the pprof HTTP server (default: false)
+|`--pprofaddr` | pprof HTTP server listening interface (default: "127.0.0.1")
```

Comment:
For the 'default:' it's best to make it match the ones above ( `<br> Default: #` ), passing html through the markdown works

Reply:
done